src/opera/commands/deploy.py

```diff
@@ -37,24 +47,50 @@ def add_parser(subparsers):
 
 
 def _parser_callback(args):
+    if args.resume and args.force:
+        print("The -r/--resume and -f/--force "
+              "options cannot be used together.")
+        return 0
```

Comment:
This is a feature of the parser and would be better implemented like that: https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group

Reply:
Thanks, now this is included.